src/traceroute/android.cpp

```diff
@@ -115,13 +116,16 @@ void AndroidProber::send_probe(std::string addr, int port, int ttl,
   measurement_kit::debug("send_probe(%s, %d, %d, %lu)", addr.c_str(), port, ttl,
              payload.length());
 
-  if (sockfd_ < 0)
-    throw std::runtime_error("Programmer error"); // already close()d
```

Comment:
Can you integrate the above comment in the exception message? It is useful to know which kind of programmer error was triggered.